src/language-html/syntax-attribute.js

```diff
@@ -18,7 +18,7 @@ function printImgSrcset(value) {
   const hasH = srcset.some(src => src.h);
   const hasX = srcset.some(src => src.d);
 
-  if (hasW + hasH + hasX !== 1) {
+  if (hasW + hasH + hasX > 1) {
     throw new Error(`Mixed descriptor in srcset is not supported`);
```

Comment:
Hm, why we use "`" here ?

Reply:
Haha, fixing

Reply:
Maybe we can do something in prettier? Fix `` ` `` to `"`

Reply:
I think we need enable the eslint `quotes` rule to avoid that in future

Reply:
Opened a issue https://github.com/prettier/prettier/issues/7301